cloud/scope/machine.go

```diff
@@ -369,6 +375,11 @@ func (m *MachineScope) Name() string {
 	return m.OCIMachine.Name
 }
 
+// IP returns the OCIMachine IP.
+func (m *MachineScope) IP() string {
+	return m.OCIMachine.Status.Addresses[0].Address
```

Comment:
What is the reason for this function instead of using https://github.com/oracle/cluster-api-provider-oci/blob/main/cloud/scope/machine.go#L410

Either way I think we may want some checks in place like the other function.

Reply:
Hi @joekr , The reason I added the `IP` function is that unlike `Network LoadBalancer`, the` LoadBalancer` does not support the "name" parameter in the `CreateBackendDetails`.

Here are the Go SDK links for both CreateBackendDetails to illustrate the differences:

LoadBalancer: https://github.com/oracle/oci-go-sdk/blob/master/loadbalancer/create_backend_details.go
Network LoadBalancer: https://github.com/oracle/oci-go-sdk/blob/master/networkloadbalancer/create_backend_details.go
Due to this difference, I used the Machine IP and PORT to construct the "backendName" for the LoadBalancer (while the Network LoadBalancer can set this name directly by using Machine Name). 

Please let me know if you have any questions or concerns about this implementation.